.changes/v2.15.0/432-features.md

```diff
@@ -0,0 +1 @@
+* Add methods `vm.ChangeCPU` and `vm.ChangeMemory` [GH-432]
```

Comment:
Please add an explanation why these changes are being made. How the new functions are better and worse.

Reply:
added